hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/DatanodeConfiguration.java

```diff
@@ -627,10 +585,50 @@ public void setWaitOnAllFollowers(boolean val) {
   )
   private long deleteContainerTimeoutMs = Duration.ofSeconds(60).toMillis();
 
+  public Duration getBlockDeletionInterval() {
+    return blockDeletionInterval;
+  }
+
+  public void setRecoveringContainerScrubInterval(Duration duration) {
+    recoveringContainerScrubInterval = duration;
+  }
+
+  public Duration getRecoveringContainerScrubInterval() {
+    return recoveringContainerScrubInterval;
+  }
+
+  public void setBlockDeletionInterval(Duration duration) {
+    blockDeletionInterval = duration;
+  }
+
+  public int getBlockDeletionLimit() {
+    return blockLimitPerInterval;
+  }
+
   public long getDeleteContainerTimeoutMs() {
     return deleteContainerTimeoutMs;
   }
 
+  public Duration getBlockDeletingMaxLockHoldingTime() {
+    return Duration.ofMillis(blockDeletingMaxLockHoldingTime);
+  }
+
+  public void setBlockDeletingMaxLockHoldingTime(Duration maxLockHoldingTime) {
+    blockDeletingMaxLockHoldingTime = maxLockHoldingTime.toMillis();
+  }
+
+  public boolean waitOnAllFollowers() {
+    return waitOnAllFollowers;
+  }
+
+  public void setWaitOnAllFollowers(boolean val) {
+    this.waitOnAllFollowers = val;
+  }
+
+  public void setBlockDeletionLimit(int limit) {
+    this.blockLimitPerInterval = limit;
+  }
```

Comment:
Since we are already moving these, how about putting `get/set` pairs next to each other, in consistent order?